Source/WebCore/css/parser/CSSPropertyParserWorkerSafe.cpp

```diff
@@ -232,13 +234,13 @@ static RefPtr<CSSValue> consumeFontFaceSrcLocal(CSSParserTokenRange& range)
     CSSParserTokenRange args = CSSPropertyParserHelpers::consumeFunction(range);
     if (args.peek().type() == StringToken) {
         auto& arg = args.consumeIncludingWhitespace();
-        if (!args.atEnd())
+        if (!args.atEnd() || !range.atEnd())
             return nullptr;
         return CSSFontFaceSrcLocalValue::create(arg.value().toAtomString());
     }
     if (args.peek().type() == IdentToken) {
         AtomString familyName = CSSPropertyParserHelpers::concatenateFamilyName(args);
-        if (familyName.isNull() || !args.atEnd())
+        if (familyName.isNull() || !args.atEnd() || !range.atEnd())
```

Comment:
Ditto

Reply:
I'll create new subtests in the wpt tests in a separated PR:
https://bugs.webkit.org/show_bug.cgi?id=250566